src/databricks/sql/client.py

```diff
@@ -763,6 +766,11 @@ def execute(
         Both will result in the query equivalent to "SELECT * FROM table WHERE field = 'foo'
         being sent to the server
 
+        async_op:
+        Denotes whether the execute command will execute the request asynchronously or not
+        By default it is set to False, if set True the execution request will be submitted and the code
+        will be non-blocking. User can later poll and request the result when ready
+
```

Comment:
I do not think we need this parameter as async will be different intereface `exec_async`